CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+0.2.1 / TBA
+==================
+- Call drain for an empty queue
+- Use NHSUK bunyan logger
+
```

Comment:
If this can be merged today the changes can become part of 0.2.0

Reply:
It's ok being part of `0.2.0` whenever it gets merged. The date of the `0.2.0` release needs to be updated to Tuesday i.e. `2017-10-31` as it hasn't been released.

Reply:
Great, I'll combine the versions and change the date.